tests/php/View/ArrayDataTest.php

```diff
@@ -11,7 +11,7 @@
 class ArrayDataTest extends SapphireTest
 {
 
-    public function testViewabledataItemsInsideArraydataArePreserved()
+    public function testViewableDataItemsInsideArraydataArePreserved()
```

Comment:
unrelated refactor that happened as part of a broader search/replace that happened to fix a small typo.